### DISCUSSION TITLE
Lesson 6 value error account only has 0

### DISCUSSION THREAD

**Discussion by gYomii** [smartcontractkit/full-blockchain-solidity-course-py#697] [Q&A] [answered]
Hi everybody, I have an issue from test with alchemy. All is running perfectly until this issue.
I do not understand what I've done wrong.
even I ad + 100000 the console returns this issue:

def test_can_fund_ans_withdraw():
        account = get_account()
        fund_me = deploy_fund_me()
        entrance_fee = fund_me.getEntranceFee() + 300000000000000
>       tx = fund_me.fund({"from": account, "value": entrance_fee})
E       ValueError: sender doesn't have enough funds to send tx. The upfront cost is: 13340543479755684 and the sender's account only has: 0

tests/test_fund_me.py:11: ValueError

**Comment by cromewar** [comment, marked as answer]
hello @gYomii it depends a lot on which network are you using to get the accounts, remember to the network to your `helpful_scripts` and also if you are using rinkeby or kovan make sure to have enough funds on your wallet to make the process. 

I hope this info might help.

**Reply by gYomii** [reply]
Thank you for your reply, I use the test comand, and I use for this command the alchemy .IO. So to my understanding, It's a fake account with normally 100 eth.
But I tried to add more fake eth to my testnet wallet and it doesn't work. I use the network fork dev. which is deployed well.
I guess the account is not funded. I checked the code and I think everything is good because it's my first real strong issue until now.
I cheked dependencies, it's seems all good,
I add + 10000000000000000 to entrace fee and it returns the same issue. So I guess it's a tiny error which is hiding in the code.

**Reply by cromewar** [reply]
Could you share your helpful scripts here please?

**Reply by gYomii** [reply]
Sure here:
```Python
from brownie import network, config, accounts, MockV3Aggregator
from web3 import Web3

FORKED_LOCAL_ENVIRONMENTS = ["mainnet-fork", "mainnet-fork-dev"]
LOCAL_BLOCKCHAIN_ENVIRONMENTS = ["development", "ganache-local"]

DECIMALS = 8
STARTING_PRICE = 200000000000


def get_account():
    if (
        network.show_active() in LOCAL_BLOCKCHAIN_ENVIRONMENTS
        or network.show_active in FORKED_LOCAL_ENVIRONMENTS
    ):
        return accounts[0]
    else:
        return accounts.add(config["wallets"]["from_key"])


def deploy_mocks():
    print(f"The active network is {network.show_active()}")
    print("Deploying Mocks...")
    if len(MockV3Aggregator) <= 0:
        MockV3Aggregator.deploy(
            DECIMALS, Web3.toWei(STARTING_PRICE, "ether"), {"from": get_account()}
        )
    print("Mocks Deployed!")

```

**Reply by cromewar** [reply]
Just a last verification did you added this networks to your brownie-config.yaml?

**Reply by gYomii** [reply]
Yes absolutly, I seted up the default either, I think there's something wrong in the account addresses when forking. But I still do not know really where the error is.

**Reply by gYomii** [reply]
yes, found it, ty so much ----> parenthesis in networ show active() in forked local env....

**Reply by cromewar** [reply]
yes sir: or network.show_active in FORKED_LOCAL_ENVIRONMENTS this was the lane getting you so much troubles, I'm glad is solved now tough.
